### DISCUSSION TITLE
Why is SVG sprite not of content type SVG?

### DISCUSSION THREAD

**Discussion by SebCorbin** [wagtail/wagtail#11649] [Development]
We are having an issue using `django-allauth` and `wagtail` because of the login page making a sprite request during login, see https://github.com/pennersr/django-allauth/issues/3643 and we're left wondering, why is the SVG icons sprite is not actual svg and just html content?

**Comment by zerolab** [comment]
@SebCorbin thank you for raising this. We allow [registering your own icons](https://docs.wagtail.org/en/stable/advanced_topics/icons.html). There are a few moving parts for mechanism that assembles the various registered icons. 

We will be looking at improving that (e.g. https://github.com/wagtail/wagtail/issues/11680) so will most definitely take this in consideration especially since it impacted other packages (e.g. https://github.com/torchbox-forks/wagtail-2fa/pull/2)
